cobalt/black_box_tests/testdata/pointer_event_test_utils.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file provides test utility functions for pointer event related
+// tests.
+
+// Fail if the test is not finished within 30 seconds.
+const kTimeout = 30 * 1000;
+const failTimer = setTimeout(fail, kTimeout);
+
+function fail() {
+    console.log("Failing due to timeout!");
+    assertTrue(false);
+}
+
+function phaseName(phase) {
+    switch (phase) {
+        case 0: return 'none';
+        case 1: return 'capturing';
+        case 2: return 'at target';
+        case 3: return 'bubbling';
+    }
+    return ' [unknown: ' + phase + ']';
+}
+
+let current_event_number = 0
+let failure_count = 0;
+
+function checkEvent(e) {
+    while (current_event_number < expected_events.length) {
+        const { name, id, phase } = expected_events[current_event_number]
+        if (name === e.name && id === e.id && phase === e.phase) {
+            break;
+        }
+        console.error(`Missing Event ['${name}', '${id}', '${phaseName(phase)}']`);
+        failure_count += 1;
```

Comment:
Instead of `+= 1`, let's do `++`
No python here :)

Reply:
Why? I thought javascript support both +=1 and ++. Is there a style guidance here?

Reply:
`++` as an incremental operator is pretty canonical, i.e. in google there's an order of magnitude more uses of that when compared to `+=`